csfieldguide/static/interactives/city-trip/README.md

```diff
@@ -0,0 +1,5 @@
+# City Trip Interactive
+
+**Author:** Courtney Bracefield, Alasdair Smith (contributed to timing logic and heapsAlgorithm.js).
+
+This interactive is created to demonstrate the travelling salesman problem.
```

Comment:
Since its for TSP perhaps it could be included again in the TSP section?

It could go just above the craypots exercise
![image](https://user-images.githubusercontent.com/25916475/57049186-09cb7380-6ccb-11e9-8400-8a4fdc824dab.png)